Core/wbDefinitionsFNV.pas

```diff
@@ -7680,50 +7679,6 @@ procedure DefineFNV;
           {0x80000000} ''
         ]);
 
-  wbPKDTType := wbEnum([
-           {0} 'Find',
-           {1} 'Follow',
-           {2} 'Escort',
-           {3} 'Eat',
-           {4} 'Sleep',
-           {5} 'Wander',
-           {6} 'Travel',
-           {7} 'Accompany',
-           {8} 'Use Item At',
-           {9} 'Ambush',
-          {10} 'Flee Not Combat',
-          {11} 'Package Type 11',
-          {12} 'Sandbox',
-          {13} 'Patrol',
-          {14} 'Guard',
-          {15} 'Dialogue',
-          {16} 'Use Weapon',
-          {17} 'Package Type 17',
-          {18} 'Combat Controller',
-          {19} 'Package Type 19',
-          {20} 'Package Type 20',
-          {21} 'Alarm',
-          {22} 'Flee',
-          {23} 'Trespass',
-          {24} 'Spectator',
```

Comment:
Looks like these were dropped in the new enumeration?

Reply:
That's because they didn't exist in the first place.  Those are not  valid package types in the CK.